### ISSUE TITLE
Cloud is Null in High score

### ISSUE THREAD

**Issue by cooktheryan** [font/pacman#8]
For some reason cloud appears as null at the high score page. It looks like the items are being directed properly

```
[POST /highscores] body = { name: 'rocok2',
  'cloud[]': 'aws',
  zone: 'us-east-2a',
  host: 'pacman-f787877d4-btv7m',
  score: '100',
  level: '1' }  host = rcook-pacman.sysdeseng.com  user-agent = Mozilla/5.0 (X11; Fedora; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/75.0.3770.100 Safari/537.36  referer = http://rcook-pacman.sysdeseng.com/
Successfully inserted highscore
Time:  Fri Jul 19 2019 18:01:35 GMT+0000 (UTC)
[GET /highscores/list]
```

**Comment by font** [comment]
@cooktheryan It looks like you have an array of `cloud[]`. The highscores API does not expect an array. The only time the `null` cloud provider can happen is if somebody dies and saves their highscore faster than their session was able to retrieve the cloud provider, which can take up to 30 seconds. If 30 seconds is hit, an `unknown` will be displayed.

**Comment by font** [comment]
@cooktheryan This may be related to some follow-up comments I made in https://github.com/font/pacman/pull/7#pullrequestreview-264456919.

**Comment by cooktheryan** [comment]
thanks @font well get you a PR next week

**Comment by dav1x** [comment]
@font @cooktheryan Added error handling and tested. Works fine. 

![Screenshot 2019-07-23 10 59 26](https://user-images.githubusercontent.com/7294149/61727489-13e49780-ad39-11e9-81ba-90a99844650b.png)

**Comment by font** [comment]
@cooktheryan @dav1x I've merged #9, built and pushed a new image `quay.io/ifont/pacman-nodejs-app:latest`. I will close this issue now, but feel free to re-open if it has not been adequately addressed. Thank you!
